.github/workflows/release.yml

```diff
@@ -1,4 +1,4 @@
-name: "Release Relayer"
+name: "(WARNING: ALPHA SOFTWARE)"
```

Comment:
Should this be prefixed with the title of the repo `Relayer`

Reply:
I guess it could, but this is appended to the version I do, so currently it'd be:

`v0.8.2 - (WARNING: ALPHA SOFTWARE)`

and this is released on this repo so the repo name feels redundant